server/plugins/DelayUGens.cpp

```diff
@@ -6210,7 +6210,7 @@ void TGrains_next(TGrains *unit, int inNumSamples)
 				if (grain->chan >= (int)numOutputs) grain->chan -= numOutputs;
 			} else {
 				grain->chan = 0;
-				pan = sc_wrap(pan * 0.5f + 0.5f, 0.f, 1.f);
+				pan = sc_clip(pan * 0.5f + 0.5f, 0.f, 1.f);
```

Comment:
There's a convention for granular ugens to mimic Pan2 for 2 channels and
PanAz for more than 2. The way I see it, TGrains tried to implement that
but flubbed the stereo math.

Backward compatibility is, however, a valid concern.